src/node/utils/Minify.js

```diff
@@ -73,7 +73,6 @@ const requestURI = (url, method, headers, callback) => {
   const parsedURL = urlutil.parse(url);
 
   let status = 500;
-  var headers = {};
```

Comment:
It's unclear to me that this is the right fix. I'd leave out this change for now until we can more completely clean up the mess that is this function.